invenio_communities/alembic/37b21951084c_update_role_id_type.py

```diff
@@ -0,0 +1,95 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Update role_id type."""
+import sqlalchemy as sa
+from alembic import op
+from invenio_accounts.alembic.f2522cdd5fcd_change_accountsrole_primary_key_to_ import (
+    downgrade as accounts_downgrade,
+)
+
+# revision identifiers, used by Alembic.
+revision = "37b21951084c"
+down_revision = "a14fa442680f"
```

Comment:
question: this revision is from [invenio-requests](https://github.com/inveniosoftware/invenio-requests/blob/18594a369a0c568b2c83a14ed9e1cb3fef447ae8/invenio_requests/alembic/a14fa442680f_create_tables.py#L15). Shouldn't it be `depends_on=a14fa442680f`?

Reply:
Fixed, was a leftover